### ISSUE TITLE
Discogs use of context in search results does not follow the pattern established in the linked data module

### ISSUE THREAD

**Issue by elrayle** [samvera/questioning_authority#278]
### Description

Discogs produces context as a hash.  The extended context for the linked data module creates the extended context as an array.

### Rationale

QA should produce consistent search results patterns to make downstream app processing the same regardless of authority accessed.

### Tested search

DISCOGS:  https://lookup.ld4l.org/authorities/search/discogs/release?q=sinatra

LINKED DATA:  https://lookup.ld4l.org/authorities/search/linked_data/agrovoc_ld4l_cache?q=milk&maxRecords=4&context=true

### Expected

First result...

```
[
  {
    "uri": "https://www.discogs.com/Frank-Sinatra-Ultimate-Sinatra/release/7121709",
    "id": "7121709",
    "label": "Frank Sinatra - Ultimate Sinatra",
    "context": [
      {
        "property": "Image URL",
        "values": [
          "https://img.discogs.com/iWcgwABw_mhcz389B0NM2Y07dhg=/fit-in/600x600/filters:strip_icc():format(jpeg):mode_rgb():quality(90)/discogs-images/R-7121709-1434203689-7495.jpeg.jpg"
        ]
      },
      {
        "property": "Year",
        "values": [
          "2015"
        ],
      },
      {
        "property": "Record Labels",
        "values": [
          "Capitol Records",
          "UMe",
          "Signature Sinatra"
        ],
      },
      {
        "property": "Formats",
        "values": [
          "Vinyl",
          "LP",
          "Compilation"
        ],
      },
      {
        "property":  "Type",
        "values": [
          "release"
        ]
      }
    ]
  },
  ...
]
```

### Actual

First results...

```
[
  {
    "uri": "https://www.discogs.com/Frank-Sinatra-Ultimate-Sinatra/release/7121709",
    "id": "7121709",
    "label": "Frank Sinatra - Ultimate Sinatra",
    "context": {
      "Image URL": [
        "https://img.discogs.com/iWcgwABw_mhcz389B0NM2Y07dhg=/fit-in/600x600/filters:strip_icc():format(jpeg):mode_rgb():quality(90)/discogs-images/R-7121709-1434203689-7495.jpeg.jpg"
      ],
      "Year": [
        "2015"
      ],
      "Record Labels": [
        "Capitol Records",
        "UMe",
        "Signature Sinatra"
      ],
      "Formats": [
        "Vinyl",
        "LP",
        "Compilation"
      ],
      "Type": [
        "values": [
          "release"
        ]
    ]
  },
  ...
]
```

**Comment by tworrall** [comment]
Pull request pending review. https://github.com/samvera/questioning_authority/pull/280

**Comment by elrayle** [comment]
Fixed by PR #280
